core-bundle/assets/controllers/module-wizard-controller.js

```diff
@@ -0,0 +1,147 @@
+import {Controller} from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['body', 'rowTemplate', 'row'];
+
+    rowSnapshots = new Map();
+
+    rowTemplateTargetConnected(template) {
+        // We need to queue a micro task here, so that Stimulus will fire
+        // rowTargetConnected().
+        queueMicrotask(() => {this._unwrap(template)});
+    }
+
+    rowTargetConnected() {
+        this._makeSortable();
+    }
+
+    rowTargetDisconnected(row) {
+        this.rowSnapshots.delete(row);
+        this._makeSortable();
+    }
+
+    copy(event) {
+        const row = this._getRow(event);
+        const snapshot = this.rowSnapshots.get(row);
+
+        row.insertAdjacentHTML('afterend', snapshot);
+        const newRow = row.nextElementSibling;
+        this.rowSnapshots.set(newRow, snapshot);
+
+        this._syncSelects(row, newRow);
+    }
+
+    delete(event) {
+        const row = this._getRow(event);
+
+        row.remove();
+    }
+
+    enable(event) {
+        event.target.previousElementSibling.checked ^= 1;
```

Comment:
Is `^=` correct here?

Reply:
Had to look it up as well, it's a bitwise XOR that toggles the checkbox:
true => false
false => true

Reply:
🦊